src/graphql/queries/ListArticles.js

```diff
@@ -153,6 +153,18 @@ export default {
             },
           }),
         },
+        articleContributesFrom: {
+          description:
+            'Show only articles with article transcript contributed by specified user',
+          type: new GraphQLInputObjectType({
```

Comment:
nit: I think we can reuse the `UserAndExistInput` above, so that we can also list articles that a person did not transcribe (so that they can focus on works they have not done yet)